docs/membership.md

```diff
@@ -0,0 +1,76 @@
+# SRT members and membership processes
+
+## Current members
+
+- Fraser Tweedale (SRT project leader; 2023-05–)
+- Gautier Di Folco (2023-05–)
+- Lei Zhu (2024-10–)
+- Mihai Maruseac (2023-05–)
+- Montez Fitzpatrick (2024-10–)
+- Tristan de Cacqueray (2023-05–)
+
+## Former members
+
+We thank past members for their valuable contributions!
+
+- Casey Mattingly (2023-05–2024-06)
+
+## Member onboarding (and offboarding)
+
+There are some necessary tasks when members join or leave the SRT.
+These are:
+
+- Update the member lists in this document.
+
+- Add (or remove) the member from the
+  `security-advisories[at]haskell.org` mailing list.  Contact the
+  [Haskell Infrastructure Admins][haskell-infra] for assistance.
+
+- Add (or remove) the member from the VINCE group, if they are
+  participating in that capacity.
+
+- Update the member list at https://www.haskell.org/security/.
+  By pull request against
+  [https://github.com/haskell-infra/www.haskell.org/](haskell-infra/www.haskell.org).
+
+- Announce the membership change(s) on [Discourse].  Usually this
+  could be included in the quarterly report.
+
+[haskell-infra]: https://github.com/haskell-infra/haskell-admins
+[Discourse]: https://discourse.haskell.org/
+
+
+## Running a Call for Volunteers
+
+To fill vacancies or grow the SRT, run a *Call for Volunteers*.
+The following is a rough guide on how we do that.
+
+- For previous calls, we appointed an SRT member to receive the
+  applications (to their personal email address).
+
+- *After applications close*, they compile the applications and
+  share with the rest of the SRT for review.  We start a voting
+  thread on the mailing list, each member states their preferred
+  applicant(s) with summary reasons, and we reach a consensus.  This
+  process has worked well, so far.
+
+- See [example content](call-for-volunteers-example.md).  There are
+  some placeholders to be filled.  The content can be modified as
+  needed.
+
+- Publish the call on [Discourse] and promote it in the logical ways
+  (e.g. Haskell Foundation social media, r/haskell, etc).
+
+- The application period should be about 4 weeks.  Bump and do
+  another burst of promotion at the halfway point.
+
+- After the application deadline, the full SRT membership reviews
+  the proposals and selects the new member(s).
```

Comment:
Would be nice to have a formal process here. Last time we used a mail thread, and I'd like us to consider a proper voting system.

Reply:
Let us plan more thoroughly ahead of the next call for volunteers (whenever that may be).